pandas/io/json/_table_schema.py

```diff
@@ -40,6 +42,8 @@
 
 loads = json.loads
 
+TABLE_SCHEMA_VERSION = "0.21.0"
```

Comment:
1.4.0

Reply:
Before I change the code further, I'd like to know why this string is called `TABLE_SCHEMA_VERSION` if it apparently should be mimicking the pandas version.